source/includes/common-minio-ad-ldap-params.rst

```diff
@@ -0,0 +1,129 @@
+
+   .. include:: /includes/common-minio-external-auth.rst
+      :start-after: start-minio-ad-ldap-server-addr
+      :end-before: end-minio-ad-ldap-server-addr
+
+   This configuration setting corresponds with the :envvar:`MINIO_IDENTITY_LDAP_SERVER_ADDR` environment variable.
+
+.. mc-cmd:: lookup_bind_dn
+   :required:
+
+   .. include:: /includes/common-minio-external-auth.rst
+      :start-after: start-minio-ad-ldap-lookup-bind-dn
+      :end-before: end-minio-ad-ldap-lookup-bind-dn
+
+   This configuration setting corresponds with the :envvar:`MINIO_IDENTITY_LDAP_LOOKUP_BIND_DN` environment variable.
+
+.. mc-cmd:: lookup_bind_password
+   :required:
+
+   .. include:: /includes/common-minio-external-auth.rst
+      :start-after: start-minio-ad-ldap-lookup-bind-password
+      :end-before: end-minio-ad-ldap-lookup-bind-password
+
+   This configuration setting corresponds with the :envvar:`MINIO_IDENTITY_LDAP_LOOKUP_BIND_PASSWORD` environment variable.
+
+.. mc-cmd:: user_dn_search_base_dn
+   :required:
+
+   .. include:: /includes/common-minio-external-auth.rst
+      :start-after: start-minio-ad-ldap-user-dn-search-base-dn
+      :end-before: end-minio-ad-ldap-user-dn-search-base-dn
+
+   This configuration setting corresponds with the :envvar:`MINIO_IDENTITY_LDAP_USER_DN_SEARCH_BASE_DN` environment variable.
+
+.. mc-cmd:: user_dn_search_filter
+   :required:
+
+   .. include:: /includes/common-minio-external-auth.rst
+      :start-after: start-minio-ad-ldap-user-dn-search-filter
+      :end-before: end-minio-ad-ldap-user-dn-search-filter
+
+   This configuration setting corresponds with the :envvar:`MINIO_IDENTITY_LDAP_USER_DN_SEARCH_FILTER` environment variable.
+
+.. mc-cmd:: comment
+   :optional:
+
+   .. include:: /includes/common-minio-external-auth.rst
+      :start-after: start-minio-ad-ldap-comment
+      :end-before: end-minio-ad-ldap-comment
+
+   This configuration setting corresponds with the :envvar:`MINIO_IDENTITY_LDAP_COMMENT` environment variable.
+
+.. mc-cmd:: enabled
+   :optional:
+
+   Set to ``false`` to disable the AD/LDAP configuration.
+
+   If ``false``, applications cannot generate STS credentials or otherwise authenticate to MinIO using the configured provider.
+
+   Defaults to ``true`` or "enabled".
+
+.. mc-cmd:: group_search_base_dn
+   :optional:
+
+   .. include:: /includes/common-minio-external-auth.rst
+      :start-after: start-minio-ad-ldap-group-search-base-dn
+      :end-before: end-minio-ad-ldap-group-search-base-dn
+
+   This configuration setting corresponds with the :envvar:`MINIO_IDENTITY_LDAP_GROUP_SEARCH_BASE_DN` environment variable.
+
+.. mc-cmd:: group_search_filter
+   :optional:
+
+   .. include:: /includes/common-minio-external-auth.rst
+      :start-after: start-minio-ad-ldap-group-search-filter
+      :end-before: end-minio-ad-ldap-group-search-filter
+
+   This configuration setting corresponds with the :envvar:`MINIO_IDENTITY_LDAP_GROUP_SEARCH_FILTER` environment variable.
```

Comment:
It's not quite a configuration setting anymore - now it's a command parameter :) 

```suggestion
   This parameter corresponds with the :envvar:`MINIO_IDENTITY_LDAP_GROUP_SEARCH_FILTER` environment variable.
```

Reply:
(a copy/paste throughout if this looks good)

Reply:
Ooo good point!